esbuild.js

```diff
@@ -41,9 +41,13 @@ const buildConfig = {
       out: "./assets/css/qld.bootstrap",
     },
     {
-      in: "./src/js/handlebars.helpers.js",
+      in: "./src/js/handlebars.init.js",
       out: "./assets/js/handlebars.helpers.bundle",
     },
+    {
+      in: "./src/js/handlebars.init.js",
+      out: "./components/handlebars.helpers.bundle",
```

Comment:
Again - does this need to be in both locations?

Reply:
the assets is what goes to cdn. the components is server side rendering. or included in the spa app. 

@stvp-qld , @MitchAnderson93 , your thoughts?

Reply:
/assets/js is good enough for Squiz use case.  Squiz will have a local copy of /assets/js on it's server due the gitbridge (cloned repo).  So it's version of server side rendering will reference handlebars jS at that path.

But for other use cases, ie if another  project wants to populate HBS variables server side, before sending to client, you'll need the helpers server side also.  Are they shipped as dependency with npm?